corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -266,10 +265,10 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
         </div>
         <!-- /ko -->
       </div>
-      <i class="hint-text" data-bind="
+      <em class="hint-text" data-bind="
```

Comment:
Why emphasis on a hint text?

Reply:
We use `<i>` pretty widely to mean "icon" and I found it slightly confusing. But you're probably right that "emphasized" isn't correct for simply italic hint text, either. I don't have incredibly strong feelings, does it seem better to leave it as `<i>`?

Reply:
I was more wondering if a screen reader uses it somehow. if now. I am fine either way

Reply:
It looks like using either `i` or `em` moves it up one level in the accessibility tree, versus using `span` which keeps it at the same level as the label text, so I opted for the latter. https://github.com/dimagi/commcare-hq/pull/34084/commits/45661a5725ad71f95f7ea5da2c1e353a43eb1298